samples/boards/reel_board/mesh_badge/src/reel_board.c

```diff
@@ -444,11 +446,7 @@ static void long_press(struct k_work *work)
 
 static bool button_is_pressed(void)
 {
-	u32_t val;
-
-	gpio_pin_read(gpio, DT_ALIAS_SW0_GPIOS_PIN, &val);
-
-	return !val;
+	return gpio_pin_get(gpio, DT_ALIAS_SW0_GPIOS_PIN);
```

Comment:
Note `gpio_pin_get` can return a negative value to indicate a failure to read a value.  There's also an essential type error here.  `gpio_pin_get(...) > 0` would be better.

Reply:
Agreed, fixed.